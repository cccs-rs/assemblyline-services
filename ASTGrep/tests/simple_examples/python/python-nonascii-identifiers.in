犺𗵰𗖣𢃵ⱝ𝐢תּ𢙝璉𓅞Ḅ𠀾𘠑齣 = 'abc'
ì·®ë»ì¾é¤ºíð½ð¾¯ãð£¶ð¡³ð¡¡®ð¢°ç¥ç¨ð²ã½Ø½Ù = 1

čćžšđ = ì·®ë»ì¾é¤ºíð½ð¾¯ãð£¶ð¡³ð¡¡®ð¢°ç¥ç¨ð²ã½Ø½Ù + 犺𗵰𗖣𢃵ⱝ𝐢תּ𢙝璉𓅞Ḅ𠀾𘠑齣

do_not_change = 17

print(čćžšđ)
print(犺𗵰𗖣𢃵ⱝ𝐢תּ𢙝璉𓅞Ḅ𠀾𘠑齣)