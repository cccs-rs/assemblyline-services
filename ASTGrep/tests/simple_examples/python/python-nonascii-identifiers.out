deobfuscated_var1 = 'abc'
deobfuscated_var2 = 1

cczsd = deobfuscated_var2 + deobfuscated_var1

do_not_change = 17

print(cczsd)
print(deobfuscated_var1)